Description: Change Firefox to Iceweasel in url handler
Author: Jesse Rhodes <drubo@drubo.net>

--- hexchat-2.10.1.orig/src/common/hexchat.c
+++ hexchat-2.10.1/src/common/hexchat.c
@@ -742,7 +742,7 @@ static char defaultconf_commands[] =
 	"NAME WII\n"			"CMD quote WHOIS %2 %2\n\n";
 
 static char defaultconf_urlhandlers[] =
-		"NAME Open Link in a new Firefox Window\n"		"CMD !firefox -new-window %s\n\n";
+		"NAME Open Link in a new Iceweasel Window\n"		"CMD !firefox -new-window %s\n\n";
 
 #ifdef USE_SIGACTION
 /* Close and open log files on SIGUSR1. Usefull for log rotating */
